app/controllers/finders_controller.rb

```diff
@@ -37,4 +38,8 @@ def facet_params
   def keywords
     params[:keywords] unless params[:keywords].blank?
   end
+
+  def atom_url
+    request.original_url.gsub('.json', '').gsub(finder_slug, "#{finder_slug}.atom")
```

Comment:
Yes, good point. I still feel like it's a bit weird to be returning the atom URL as part of the JSON response. Maybe it's fine!

If we do want to keep it in the JSON, is there any prior art for this on GOV.UK? Perhaps something along the lines of HTML's `<link rel="alternate" />`? Since we're basically saying, "here's a place you can get this same content in a different format".